source/contracts/reporting/ReportingWindow.sol

```diff
@@ -34,16 +36,22 @@ contract ReportingWindow is DelegationTarget, Typed, Initializable, IReportingWi
     Set.Data private markets;
     Set.Data private limitedReporterMarkets;
     Set.Data private allReporterMarkets;
+    uint256 private indeterminateMarketCount;
     mapping(address => ReportingStatus) private reporterStatus;
     mapping(address => uint256) private numberOfReportsByMarket;
     uint256 private constant BASE_MINIMUM_REPORTERS_PER_MARKET = 7;
+    RunningAverage.Data private reportingGasPrice;
+    RunningAverage.Data private marketReports;
 
     function initialize(IUniverse _universe, uint256 _reportingWindowId) public beforeInitialized returns (bool) {
         endInitialization();
         universe = _universe;
         startTime = _reportingWindowId * universe.getReportingPeriodDurationInSeconds();
         RegistrationTokenFactory _registrationTokenFactory = RegistrationTokenFactory(controller.lookup("RegistrationTokenFactory"));
         registrationToken = _registrationTokenFactory.createRegistrationToken(controller, this);
+        // Initialize these to some reasonable value to handle the first market ever created without branching code 
+        reportingGasPrice.record(Reporting.defaultReportingGasPrice());
+        marketReports.record(Reporting.defaultReportsPerMarket());
```

Comment:
Are we OK with these weighing into the final averages?  Is the assumption that we should have enough markets to drown out any amount of outlying them may trigger?

Reply:
My thoughts on this are that since its just an estimate and it lets us avoid branching code in oft used getters its alright. I could be easily convinced otherwise though